vocabularies/HTML5.xml

```diff
@@ -88,6 +88,14 @@
         </Annotation>
       </TypeDefinition>
 
+      <Term Name="MergeDuplicateValuesInColumns" Type="Collection(Edm.PropertyPath)" Nullable="false">
```

Comment:
I find the "in columns" misleading as this is row-span, so vertically merge adjacent cells with the same value in one columns.

Reply:
Do you have a better name - I changed it definitely 5 times before pushing :-) Currently I try to say that it merges duplicate values in a column - adjacent is missing - but simply adding it, makes the name even longer.